_pyceres/core/crs_matrix.h

```diff
@@ -13,14 +13,21 @@ namespace py = pybind11;
 
 namespace {
 py::tuple ConvertCRSToPyTuple(const ceres::CRSMatrix& crsMatrix) {
-  std::vector<int> rows, cols;
-  std::vector<double> values;
+  size_t n_values = crsMatrix.values.size();
+  py::array_t<int> rows(n_values), cols(n_values);
+  py::array_t<double> values(n_values);
 
+  int* rows_data = static_cast<int*>(rows.request().ptr);
+  int* cols_data = static_cast<int*>(cols.request().ptr);
+  double* values_data = static_cast<double*>(values.request().ptr);
```

Comment:
All these pointers can be marked as `const`?

Reply:
We will need to update the arrays so we cannot make the pointers const?

Reply:
Done.